EnergyCorrelatorOptions.h

```diff
@@ -60,10 +60,14 @@ namespace EnergyCorrelatorOptions {
 
   // Smearing options
   const bool modCsts = true;
-  const float theta = 1;
-  const float pTSmear = 0;
+  const float theta = 0;
+  const float pTSmear = 0.02;
   const float effVal = 1;
 
+  //Manual calculation options
+  const bool manualTwoPoint = true;
+  const bool manualThreePoint = false;
+
```

Comment:
Notice how `nPoints` is a vector: my intent is to (soon) add the ability to calculate multiple nPoint per run of the module. So I would suggest instead have 1 flag (`doManualCalc`), and then based on what `nPoint`s are specified, the corresponding manual calculations would be run!

```suggestion
  //Manual calculation options
  const bool doManualCalc = true;

```